module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -245,8 +246,29 @@ protected function parseConfig($formId, $config)
                 }
                 if (isset($el['options'])) {
                     $options = [];
+                    $placeholder
+                        = $elementType === 'select'
+                        ? ($element['placeholder'] ?? null)
+                        : null;
+
+                    if ($placeholder) {
+                        $options[] = [
+                            'value' => '',
+                            'label' => $this->translate($placeholder),
+                            'attributes' => [
+                                'selected' => 'selected', 'disabled' => 'disabled'
+                            ]
+                        ];
+                    }
                     foreach ($el['options'] as $option) {
-                        $options[$option] = $this->translate($option);
+                        if ($placeholder) {
+                            $options[] = [
```

Comment:
I think it's a little confusing that we format $options differently depending on whether or not there is a placeholder. Is this necessary for a particular reason? Would it be better to always format $options in the more verbose way just for consistency/readability? If I'm missing something and we have to do it like this, a comment explaining why would be helpful, so it doesn't get factored out by some future developer. :_)

Reply:
Yes, I agree. Refactored to use the more verbose way for select element options also when `placeholder` is not used.

Reply:
Thanks!